### PR TITLE
[homeconnect] Update README.md

### PR DESCRIPTION
**Pull request by Linsenschisdre**
Added full set of items for dishwasher, oven and washer.

I find it quite inconvenient to use the provided table in order so build up my .items file for my appliances. As I have a dishwasher, an oven and a washer in my home and prepared the whole items stack anyway, I decided to let other users make use of it and save them some time by just "copy & paste" what they need.

### CODE REVIEW

File: bundles/org.openhab.binding.homeconnect/README.md [hunk 1]
```diff
@@ -161,15 +161,60 @@ The channel parameter uses the following syntax: `homeconnect:<thing type id>:<b
...
+String                 Dishwasher_SelectedProgramState                    "Selected Program"                   {channel="homeconnect:dishwasher:api_bridge_at_home:dishwasher1:selected_program_state"}
+String                 Dishwasher_ActiveProgramState                      "Active Program"                     {channel="homeconnect:dishwasher:api_bridge_at_home:dishwasher1:active_program_state"}
+Number:Time            Dishwasher_RemainingProgramTimeState               "Remaining program time"             {channel="homeconnect:dishwasher:api_bridge_at_home:dishwasher1:remaining_program_time_state"}
+Number:Dimensionless   Dishwasher_ProgramProgressState                    "Progress State"                     {channel="homeconnect:dishwasher:api_bridge_at_home:dishwasher1:program_progress_state"}
+String                 Dishwasher_basic_actions_state                     "Basic control"                      {channel="homeconnect:dishwasher:api_bridge_at_home:sgeschirrspuelmaschine:basic_actions_state"}
```
Comment (lsiepel) [inline bundles/org.openhab.binding.homeconnect/README.md:173]:
```suggestion
String                 Dishwasher_BasicActionsState                     "Basic Control"                      {channel="homeconnect:dishwasher:api_bridge_at_home:sgeschirrspuelmaschine:basic_actions_state"}
```

File: bundles/org.openhab.binding.homeconnect/README.md [hunk 2]
```diff
@@ -161,15 +161,60 @@ The channel parameter uses the following syntax: `homeconnect:<thing type id>:<b
...
+Number:Dimensionless   Oven_ProgramProgressState                          "Progress State"                     {channel="homeconnect:oven:api_bridge_at_home:oven1:program_progress_state"}
+Number:Time            Oven_elapsed_program_time                          "Elapsed program time"               {channel="homeconnect:oven:api_bridge_at_home:oven1:elapsed_program_time"}
+Number:Time            Oven_duration                                      "Program duration"                   {channel="homeconnect:oven:api_bridge_at_home:oven1:duration"}
+Number:Temperature     Oven_oven_current_cavity_temperature               "Current temperature"                {channel="homeconnect:oven:api_bridge_at_home:oven1:oven_current_cavity_temperature"}
+Number:Temperature     Oven_setpoint_temperature                          "Set temperature"                    {channel="homeconnect:oven:api_bridge_at_home:oven1:setpoint_temperature"}
+String                 Oven_basic_actions_state                           "Basic control"                      {channel="homeconnect:oven:api_bridge_at_home:oven1_:basic_actions_state"}
```
Comment (lsiepel) [inline bundles/org.openhab.binding.homeconnect/README.md:189]:
```suggestion
Number:Time            Oven_ElapsedProgramTime                          "Elapsed Program Time"               {channel="homeconnect:oven:api_bridge_at_home:oven1:elapsed_program_time"}
Number:Time            Oven_Duration                                      "Program Duration"                   {channel="homeconnect:oven:api_bridge_at_0home:oven1:duration"}
Number:Temperature     Oven_CurrentCavityTemperature               "Current Temperature"                {channel="homeconnect:oven:api_bridge_at_home:oven1:oven_current_cavity_temperature"}
Number:Temperature     Oven_SetpointTemperature                          "Set Temperature"                    {channel="homeconnect:oven:api_bridge_at_home:oven1:setpoint_temperature"}
String                 Oven_BasicActionsState                           "Basic Control"                      {channel="homeconnect:oven:api_bridge_at_home:oven1_:basic_actions_state"}
```

File: bundles/org.openhab.binding.homeconnect/README.md [hunk 3]
```diff
@@ -161,15 +161,60 @@ The channel parameter uses the following syntax: `homeconnect:<thing type id>:<b
...
+Switch                 Washer_laundry_care_washer_rinse_hold              "Spin hold"                          {channel="homeconnect:washer:api_bridge_at_home:washer1:laundry_care_washer_rinse_hold"}
+Switch                 Washer_laundry_care_washer_soak                    "Soaking state"                      {channel="homeconnect:washer:api_bridge_at_home:washer1:laundry_care_washer_soak"}
+Number:Mass            Washer_laundry_care_washer_load_recommendation     "Load recommendation"                {channel="homeconnect:washer:api_bridge_at_home:washer1:laundry_care_washer_load_recommendation"}
+Number:Dimensionless   Washer_program_energy                              "Estimated energy consumption"       {channel="homeconnect:washer:api_bridge_at_home:washer1:program_energy"}
+Number:Dimensionless   Washer_program_water                               "Estimated water consumption"        {channel="homeconnect:washer:api_bridge_at_home:washer1:program_water"}
+String                 Washer_basic_actions_state                         "Basic control"                      {channel="homeconnect:washer:api_bridge_at_home:washer1:basic_actions_state"}
```
Comment (lsiepel) [inline bundles/org.openhab.binding.homeconnect/README.md:216]: similar comments as previous, please use consistent naming strategy, so it is a true copy paste solution.

**Review by lsiepel — CHANGES_REQUESTED** [review]
Otherwise LGTM

**Comment by lsiepel** [comment]
Minor request: would be nice if the example could be vertically aligned.

File: bundles/org.openhab.binding.homeconnect/README.md [hunk 4]
```diff
@@ -161,15 +161,60 @@ The channel parameter uses the following syntax: `homeconnect:<thing type id>:<b
...
+String                 Dishwasher_SelectedProgramState                    "Selected Program"                   {channel="homeconnect:dishwasher:api_bridge_at_home:dishwasher1:selected_program_state"}
+String                 Dishwasher_ActiveProgramState                      "Active Program"                     {channel="homeconnect:dishwasher:api_bridge_at_home:dishwasher1:active_program_state"}
+Number:Time            Dishwasher_RemainingProgramTimeState               "Remaining Program Time"             {channel="homeconnect:dishwasher:api_bridge_at_home:dishwasher1:remaining_program_time_state"}
+Number:Dimensionless   Dishwasher_ProgramProgressState                    "Progress State"                     {channel="homeconnect:dishwasher:api_bridge_at_home:dishwasher1:program_progress_state"}
+String                 Dishwasher_BasicActionsState                     "Basic Control"                      {channel="homeconnect:dishwasher:api_bridge_at_home:sgeschirrspuelmaschine:basic_actions_state"}
```
Comment (jlaur) [inline bundles/org.openhab.binding.homeconnect/README.md:173]:
```suggestion
String                 Dishwasher_BasicActionsState                       "Basic Control"                      {channel="homeconnect:dishwasher:api_bridge_at_home:dishwasher1:basic_actions_state"}
```

File: bundles/org.openhab.binding.homeconnect/README.md [hunk 5]
```diff
@@ -161,15 +161,60 @@ The channel parameter uses the following syntax: `homeconnect:<thing type id>:<b
...
+String                 Oven_SelectedProgramState                          "Selected Program"                   {channel="homeconnect:oven:api_bridge_at_home:oven1:selected_program_state"}
+String                 Oven_ActiveProgramState                            "Active Program"                     {channel="homeconnect:oven:api_bridge_at_home:oven1:active_program_state"}
+Number:Time            Oven_RemainingProgramTimeState                     "Remaining Program Time"             {channel="homeconnect:oven:api_bridge_at_home:oven1:remaining_program_time_state"}
+Number:Dimensionless   Oven_ProgramProgressState                          "Progress State"                     {channel="homeconnect:oven:api_bridge_at_home:oven1:program_progress_state"}
+Number:Time            Oven_ElapsedProgramTime                          "Elapsed Program Time"               {channel="homeconnect:oven:api_bridge_at_home:oven1:elapsed_program_time"}
+Number:Time            Oven_Duration                                      "Program Duration"                   {channel="homeconnect:oven:api_bridge_at_0home:oven1:duration"}
```
Comment (jlaur) [inline bundles/org.openhab.binding.homeconnect/README.md:186]:
```suggestion
Number:Time            Oven_Duration                                      "Program Duration"                   {channel="homeconnect:oven:api_bridge_at_home:oven1:duration"}
```

File: bundles/org.openhab.binding.homeconnect/README.md [hunk 6]
```diff
@@ -161,15 +161,60 @@ The channel parameter uses the following syntax: `homeconnect:<thing type id>:<b
...
+Switch                 Oven_RemoteControlActiveState                      "Remote Control Activation State"    {channel="homeconnect:oven:api_bridge_at_home:oven1:remote_control_active_state"}
+String                 Oven_SelectedProgramState                          "Selected Program"                   {channel="homeconnect:oven:api_bridge_at_home:oven1:selected_program_state"}
+String                 Oven_ActiveProgramState                            "Active Program"                     {channel="homeconnect:oven:api_bridge_at_home:oven1:active_program_state"}
+Number:Time            Oven_RemainingProgramTimeState                     "Remaining Program Time"             {channel="homeconnect:oven:api_bridge_at_home:oven1:remaining_program_time_state"}
+Number:Dimensionless   Oven_ProgramProgressState                          "Progress State"                     {channel="homeconnect:oven:api_bridge_at_home:oven1:program_progress_state"}
+Number:Time            Oven_ElapsedProgramTime                          "Elapsed Program Time"               {channel="homeconnect:oven:api_bridge_at_home:oven1:elapsed_program_time"}
```
Comment (jlaur) [inline bundles/org.openhab.binding.homeconnect/README.md:185]:
```suggestion
Number:Time            Oven_ElapsedProgramTime                            "Elapsed Program Time"               {channel="homeconnect:oven:api_bridge_at_home:oven1:elapsed_program_time"}
```

File: bundles/org.openhab.binding.homeconnect/README.md [hunk 7]
```diff
@@ -161,15 +161,60 @@ The channel parameter uses the following syntax: `homeconnect:<thing type id>:<b
...
+Number:Time            Oven_Duration                                      "Program Duration"                   {channel="homeconnect:oven:api_bridge_at_0home:oven1:duration"}
+Number:Temperature     Oven_CurrentCavityTemperature               "Current Temperature"                {channel="homeconnect:oven:api_bridge_at_home:oven1:oven_current_cavity_temperature"}
```
Comment (jlaur) [inline bundles/org.openhab.binding.homeconnect/README.md:187]:
```suggestion
Number:Temperature     Oven_CurrentCavityTemperature                     "Current Temperature"                {channel="homeconnect:oven:api_bridge_at_home:oven1:oven_current_cavity_temperature"}
```

File: bundles/org.openhab.binding.homeconnect/README.md [hunk 8]
```diff
@@ -161,15 +161,60 @@ The channel parameter uses the following syntax: `homeconnect:<thing type id>:<b
...
+Number:Temperature     Oven_SetpointTemperature                          "Set Temperature"                    {channel="homeconnect:oven:api_bridge_at_home:oven1:setpoint_temperature"}
+String                 Oven_BasicActionsState                           "Basic Control"                      {channel="homeconnect:oven:api_bridge_at_home:oven1_:basic_actions_state"}
```
Comment (jlaur) [inline bundles/org.openhab.binding.homeconnect/README.md:189]:
```suggestion
String                 Oven_BasicActionsState                            "Basic Control"                      {channel="homeconnect:oven:api_bridge_at_home:oven1:basic_actions_state"}
```

**Review by lsiepel — APPROVED** [review]
LGTM

**Pull request merged**